greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-6592: Add minimal navigation option to Action page type

Ref: https://jira.greenpeace.org/browse/PLANET-6592

> we want to provide the option for the minimal navigation, same as we currently have on Campaigns.

![Screenshot from 2022-02-23 18-18-07](https://user-images.githubusercontent.com/617346/155372037-0a0a3ac3-905b-4828-b06f-b5c16c2de38e.png)

## Test

- Activate _Action_ pagetype in _Planet 4 > Information Architecture_
- Add an Action page
- Edit the Navigation type in the sidebar, check that the proper navigation bar is displayed according to the configuration
